packages/amazonq/src/extension.ts

```diff
@@ -141,32 +143,48 @@ export async function activateAmazonQCommon(context: vscode.ExtensionContext, is
         }, 1000)
     }
 
-    await telemetry.auth_userState.run(async () => {
-        telemetry.record({ passive: true })
-
-        const firstUse = AuthUtils.ExtensionUse.instance.isFirstUse()
-        const wasUpdated = AuthUtils.ExtensionUse.instance.wasUpdated()
-
-        if (firstUse) {
-            telemetry.record({ source: ExtStartUpSources.firstStartUp })
-        } else if (wasUpdated) {
-            telemetry.record({ source: ExtStartUpSources.update })
-        } else {
-            telemetry.record({ source: ExtStartUpSources.reload })
-        }
-
-        const authState = (await AuthUtil.instance.getChatAuthState()).codewhispererChat
-        const currConn = AuthUtil.instance.conn
-        if (currConn !== undefined && !isAnySsoConnection(currConn)) {
-            getLogger().error(`Current Amazon Q connection is not SSO, type is: %s`, currConn?.type)
-        }
-
-        telemetry.record({
-            authStatus: authState === 'connected' || authState === 'expired' ? authState : 'notConnected',
-            authEnabledConnections: AuthUtils.getAuthFormIdsFromConnection(currConn).join(','),
-            ...(await getTelemetryMetadataForConn(currConn)),
+    try {
+        await telemetry.auth_userState.run(async () => {
```

Comment:
nit: typically (I'd even call it idiomatic) we just add `.catch()` if we want to handle `run()` exceptions. Avoids extra nesting.

```suggestion
    try {
        await telemetry.auth_userState.run(async () => { ... }).catch(...)
```